src/I18n/Translator.php

```diff
@@ -33,7 +35,18 @@ class Translator extends BaseTranslator
      */
     public function translate($key, array $tokensValues = [])
     {
-        $message = $this->getMessage($key);
+        if (isset($tokensValues['_count'])) {
+            $message = $this->getMessage(static::PLURAL_PREFIX . $key);
+            if (!$message) {
+                $message = $this->getMessage($key);
+            }
```

Comment:
We'll need this for backwards compatibilty and userland defined message loaders/parsers.

Reply:
What exactly do you mean? Do I need to change something else here?

Reply:
No. I was just commenting on it for future us.